src/modExcelToXML.bas

```diff
@@ -0,0 +1,169 @@
+Attribute VB_Name = "modExcelToXML"
+Option Explicit
+
+Const ms As Double = 0.000000011574
+
+Sub excelToXML()
+    
+    If Not changeFileExtension(ThisWorkbook.Path, ThisWorkbook.Name) Then
+            
+    End If
+        
+    If Not windowsUnZip(Left(ThisWorkbook.FullName, Len(ThisWorkbook.FullName) - 4) & "zip", ThisWorkbook.Path) Then
+    
+    End If
+        
+End Sub
+
+Function changeFileExtension(strPath As String, strFileName As String) As Boolean
+    
+    Dim FSO As New FileSystemObject
+    Dim strBaseName As String
+    Dim strExtension As String
+
+    On Error GoTo errHandler
+    
+    If Right(strPath, Len(strPath) - 1) <> Application.PathSeparator Then
+        strPath = strPath & Application.PathSeparator
+    End If
+    
+    strBaseName = FSO.GetBaseName(strPath & strFileName)
+    strExtension = FSO.GetExtensionName(strPath & strFileName)
+    
+    Call FSO.CopyFile(strPath & strFileName, strPath & strBaseName & ".zip")
```

Comment:
FSO.BuildPath is a robust way to concatenate paths. It takes care of extra or missing slashes, etc.